backend/src/main/scala/cromwell/backend/validation/MaxRetriesValidation.scala

```diff
@@ -0,0 +1,43 @@
+package cromwell.backend.validation
+
+import cats.syntax.validated._
+import com.typesafe.config.Config
+import common.validation.ErrorOr.ErrorOr
+import wom.RuntimeAttributesKeys._
+import wom.types.WomIntegerType
+import wom.values.{WomInteger, WomValue}
+
+/**
+  * Validates the "maxRetries" runtime attribute an Integer greater than 0, returning the value as an `Int`.
+  *
+  * `instance` returns an validation that errors when no attribute is specified.
+  *
+  * `default` a hardcoded default WomValue for MaxRetries.
+  *
+  * `configDefaultWomValue` returns the value of the attribute as specified by the
+  * reference.conf file, coerced into a WomValue.
+  */
+object MaxRetriesValidation {
+  lazy val instance: RuntimeAttributesValidation[Int] = new MaxRetriesValidation(MaxRetriesKey)
+  lazy val optional: OptionalRuntimeAttributesValidation[Int] = instance.optional
+
+  def default(runtimeConfig: Option[Config]): RuntimeAttributesValidation[Int] = instance.withDefault(
+    configDefaultWomValue(runtimeConfig) getOrElse WomInteger(0))
+  def configDefaultWomValue(config: Option[Config]): Option[WomValue] = instance.configDefaultWomValue(config)
+  //maybe this needs a cieling declared? It could be detrimental....
```

Comment:
i before e except after c

Reply:
Since nobody else mentioned a ceiling is required, I'll go ahead and delete the comment!